docs/03_server/02_data-server/03_advanced.md

```diff
@@ -217,6 +217,97 @@ are always selected in the order they are specified in the index.
 
 The `index` property can have unique and non-unique [indexes](../../../database/data-types/index-entities/#types)
 
+Examples when: `numKeyFields > 1`
+
+Range is based on sorted records when numKeyFields is greater than 1
+For below table and data which is sorted already, we have shown a couple of examples of ranged query with its result
```

Comment:
I would rephrase it like:

"Let's see a couple of examples of ranged queries with their respective results in CSV format. The table definition with associated data sorted by its primary key fields is shown below.